github/ci/prow-deploy/kustom/base/manifests/local/github-jira-proxy-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: github-jira-proxy
+  labels:
+    app: github-jira-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: github-jira-proxy
+  template:
+    metadata:
+      labels:
+        app: github-jira-proxy
+    spec:
+      terminationGracePeriodSeconds: 180
+      containers:
+        - name: github-jira-proxy
+          image: quay.io/kubevirtci/github-jira-proxy:latest
+          args:
+            - --dry-run=false
+            - --port=9900
+            - --github-token-path=/etc/github/oauth
+            - --github-endpoint=http://ghproxy
+            - --github-endpoint=https://api.github.com
+          ports:
+            - name: http
+              containerPort: 9900
+          volumeMounts:
```

Comment:
no need

Reply:
create it here: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/tasks/secrets.yml